### PR TITLE
Chore/enhance stacked list

### PR DESCRIPTION
**Pull request by dawntancarrasco**
Changes:
1. Added some properties to input and text area, and stacked list
2. changed select functionality on tabs
3. added some tests and stories for the compoents

Screenshot 2023-10-20 at 1 00 00 PM

### CODE REVIEW

File: src/components/Input/Input.styles.ts [hunk 1]
```diff
@@ -80,7 +142,9 @@ const StyledInput = styled.input<{
 
   &:disabled{
     background-color:${theme.core.background.bgSecondary};
+    color${theme.core.content.contentTertiary}
```
Comment (sbracegirdle) [inline src/components/Input/Input.styles.ts:145]: Question; is there meant to be a colon between `color` and the colour value ?

Comment (sbracegirdle) [reply src/components/Input/Input.styles.ts:145]: Is there meant to be a semi-colon at end of expression?

**Review by sbracegirdle — APPROVED** [review]
Looks good, just one question about possible typo in colour styling rule

File: src/components/Input/Input.styles.ts (hunk 1 above)
Comment (dawntancarrasco) [reply src/components/Input/Input.styles.ts]: yup fixed it! :) thanks!

**Pull request merged**